src/Tests/Utils/Helpers.test.ts

```diff
@@ -156,5 +157,22 @@ suite('Utils', function() {
         assert.isTrue(RealPath.exists('/'));
       });
     });
+
+    suite('#isOneExplorerTargetFile()', function() {
+      test('check if it is target file of OneExplorer', function() {
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.pb')));
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.onnx')));
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.tflite')));
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.circle')));
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.cfg')));
+        assert.isTrue(isOneExplorerTargetFile(vscode.Uri.file('test.log')));
+      });
+      test('NEG: it is not target file of OneExplorer', function() {
+        assert.isFalse(isOneExplorerTargetFile(vscode.Uri.file('test.any')));
+        assert.isFalse(isOneExplorerTargetFile(vscode.Uri.file('.onnx/')));
+        assert.isFalse(isOneExplorerTargetFile(vscode.Uri.file('onnx')));
+        assert.isFalse(isOneExplorerTargetFile(vscode.Uri.file('')));
```

Comment:
'' and '.onnx/' can be negative tests, but 'test.any' is positive, because it's the function's expected falsy input.
You can include 'test.any' to the above test, but not the others.
- '' : NEG: check empty path
- '.onnx/': NEG: check directory
- 'onnx': NEG: check a file without ext

Reply:
thanks!